examples/basic/README.md

```diff
@@ -0,0 +1,55 @@
+# Basic
+
+This example shows how to use the library with the minimal properties.
```

Comment:
Miniumum example, here you can just drag the focal point and the example pictures focal point will be updated on real time